cmd/createtree/main_test.go

```diff
@@ -195,6 +186,25 @@ func runTest(t *testing.T, tests []*testCase) {
 	}
 }
 
+// expectCalls returns the minimum number of times a function is expected to be called
+// given the return error for the function (err), and all previous errors in the function's
+// code path.
+func expectCalls(err error, prevErr ...error) int {
+	// If a function prior to this function errored,
+	// we do not expect this function to be called.
+	for _, e := range prevErr {
+		if e != nil {
+			return 0
+		}
+	}
+	// If this function errors, it will be retried multiple times.
+	if err != nil {
+		return 3
```

Comment:
Should this just be 1 (or perhaps 2 if we're testing that it does retry)? and let the `MinTimes` take care of it?

Reply:
Yep. I'm still using this in `MinTimes` I can make the number 2